java/ql/src/semmle/code/java/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -46,7 +46,10 @@ class ReturnNode extends ExprNode {
 
 /** A data flow node that represents a call. */
```

Comment:
Needs an update. The node doesn't really represent a call - it's a value being returned from a call.